sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeSentimentOptions.java

```diff
@@ -49,6 +49,20 @@ public AnalyzeSentimentOptions setIncludeStatistics(boolean includeStatistics) {
         return this;
     }
 
+    /**
+     * Set the value of {@code disableServiceLogs}.
+     *
+     * @param disableServiceLogs If a boolean value was specified in the request this field will contain
+     * logging information about the document payload.
+     *
+     * @return the {@link AnalyzeSentimentOptions} object itself.
```

Comment:
I am not sure if this comment is true. What field?

Reply:
Good catch! It is the docstring for include statistics